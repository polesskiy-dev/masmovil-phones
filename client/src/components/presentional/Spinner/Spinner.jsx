import React from 'react';

import './Spinner.css';

const Spinner = () => (
  <div className="spinner-lds-pacman__wrapper">
    <div className="spinner-lds-pacman">
      <div>
        <div />
        <div />
        <div />
      </div>
      <div>
        <div />
        <div />
      </div>
    </div>
  </div>
);

export default Spinner;
