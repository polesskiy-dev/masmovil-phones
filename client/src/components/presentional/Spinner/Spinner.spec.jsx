import React from 'react';
import renderer from 'react-test-renderer';

import Spinner from './Spinner';

it('<Spinner/ > renders correctly', () => {
  const tree = renderer.create(<Spinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
