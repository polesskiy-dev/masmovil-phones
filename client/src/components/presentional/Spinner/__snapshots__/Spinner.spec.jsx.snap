// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/ > renders correctly 1`] = `
<div
  className="spinner-lds-pacman__wrapper"
>
  <div
    className="spinner-lds-pacman"
  >
    <div>
      <div />
      <div />
      <div />
    </div>
    <div>
      <div />
      <div />
    </div>
  </div>
</div>
`;
